docs/event_handlers/http.md

```diff
@@ -0,0 +1,70 @@
+# HTTP Webhook Event Handling
+
+While it's not neccesary for every Saleor app to receive domain events from Saleor it is possible, as described in [:saleor-saleor: Saleor's docs](https://docs.saleor.io/docs/3.0/developer/extending#apps).
+
+To configure your app to listen to HTTP webhooks issued from Saleor you need to provide your app with two more arguments.
+
+## Setting up the Saleor App
+
+### Defining webhook handlers
+
+An HTTP webhook handler is a function that is exactly ile one that one would use as a FastAPI endpoint. There is no need to register those as routes since that is deatl with when the app is initialized.
```

Comment:
`exactly ile` ? 😄

Reply:
`deatl` ?

Reply:
wow... those are some 2am typos (if my hands were those of a rhino...)